motoko/canister_logs/src/Main.mo

```diff
@@ -0,0 +1,14 @@
+import Debug "mo:base/Debug";
+
+actor CanisterLogs {
+
+  public func print(text : Text) : async () {
+    Debug.print(text);
+  };
+
+  public func trap(text : Text) : async () {
+    Debug.print("right before trap");
+    Debug.trap(text);
+  };
+
```

Comment:
@crusso could you please advice on how to implement the timer here (with the same functionality from rust example)? I tried to use an example from the docs but it failed to compile.